src/middleware/logToDB.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from 'express';
+import { LogEntry } from '../models/logEntry';
+import jwt from 'jsonwebtoken';
+import { TokenInterface } from '../classes/token';
+
+
+export async function logToDB(req: Request, res: Response, next: NextFunction) {
+    const now= new Date();
```

Comment:
Let's keep the handler simple and move all the implementation details to a controller typescript class.

Reply:
done